packages/core/directives/marker.ts

```diff
@@ -158,8 +158,11 @@ export class AgmMarker implements OnDestroy, OnChanges, AfterContentInit {
 
   /** @internal */
   ngOnChanges(changes: {[key: string]: SimpleChange}) {
-    if (typeof this.latitude !== 'number' || typeof this.longitude !== 'number') {
-      return;
+    if (typeof this.latitude === 'string') {
```

Comment:
this breaks the case that the marker only gets added to the manager if latitude/longitude are numbers. Can you add these check underneath?

Reply:
My bad! I added the check again